std_msgs/Header header
common/Uint8TimeStamped cdcu_bodycmdperd_err
common/Uint8TimeStamped cdcu_bodycmddrop_err
common/Uint8TimeStamped cdcu_bodycmdchksum_err
common/Uint8TimeStamped cdcu_bodycmdoffline_err
common/Uint8TimeStamped cdcu_headlampopencirct_err
common/Uint8TimeStamped cdcu_headlampshrtcirct_err
common/Uint8TimeStamped cdcu_turnllampopencirct_err
common/Uint8TimeStamped cdcu_turnllampshrtcirct_err
common/Uint8TimeStamped cdcu_turnrlampopencirct_err
common/Uint8TimeStamped cdcu_turnrlampshrtcirct_err
common/Uint8TimeStamped cdcu_backlampopencirct_err
common/Uint8TimeStamped cdcu_backlampshrtcirct_err
common/Uint8TimeStamped cdcu_buzzeropencirct_err
common/Uint8TimeStamped cdcu_buzzershrtcirct_err
common/Uint8TimeStamped cdcu_hornopencirct_err
common/Uint8TimeStamped cdcu_hornshrtcirct_err
common/Uint8TimeStamped cdcu_runlampopencirct_err
common/Uint8TimeStamped cdcu_runlampshrtcirct_err
common/Uint8TimeStamped cdcu_brklampopencirct_err
common/Uint8TimeStamped cdcu_brklampshrtcirct_err
